packages/evolution-generator/src/tests/test_generate_widgets.py

```
@@ -1278,10 +1287,221 @@ def test_returns_none_when_group_does_not_match_row_group(self):
+        row["help_popup"],
+        row["conditional"],
+        row["validation"],
+        widget_label,
+        row,
+    )
+
+    assert "helpPopup: customHelpPopup.householdLanguageHelpPopup" in code
+
+
+@pytest.mark.parametrize(
+    "generator,args,help_popup_name",
```

Ruff (0.16.1)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)